requirements.txt

```diff
@@ -0,0 +1,5 @@
+.
+pytest
+nbval
+umap-learn
+plotnine
```

Comment:
Are pandas, numpy, etc not required?

Typically a requirements file has more detail about what is required, like in an evironment.yml file. (See [this link](https://packaging.python.org/discussions/install-requires-vs-requirements/#requirements-files) for more details.

Reply:
Also, if you have different requirements for installing and testing ponyo you can use the `tests_require` parameter [shown here](https://stackoverflow.com/a/15422703/10930590) and install ponyo with `pip install .[test]`

Reply:
Good point! That is confusing. The `requirements.txt` was only meant to indicate requirements for testing and the install requirements are in the `setup.py` file. 

I will update my setup.py accordingly